frontend/packages/cli/vite-plugins/set-env.ts

```diff
@@ -40,19 +40,38 @@ export function setEnvPlugin(): Plugin {
     }
   }
 
-  const versionPrefix = '@liam-hq/cli@'
+  const versionPrefix = 'refs/tags/@liam-hq/cli@'
 
   const isReleasedGitHash = (gitHash: string, packageJsonVersion: string) => {
     const latestTagName = `${versionPrefix}${packageJsonVersion}`
     try {
       execSync('git fetch --tags')
-      const tagCommit = execSync(`git rev-parse ${latestTagName}`)
+
+      const tagList = execSync('git tag -l').toString()
+      // Get the tag from remote because it cannot be obtained by automatic deployment of Vercel.
+      if (!tagList.includes(latestTagName)) {
+        const lsRemoteOutput = execSync(
+          'git ls-remote --tags https://github.com/liam-hq/liam.git',
+        )
+          .toString()
+          .trim()
+        const tagCommit = lsRemoteOutput
+          .split('\n')
+          .find((line) => line.includes(latestTagName))
```

Comment:
💭   This comparison feels a bit naive for two reasons:  

- `refs/tags/@liam-hq/cli@0.4.1` might match both `refs/tags/@liam-hq/cli@0.4.1` and `refs/tags/@liam-hq/cli@0.4.1{}`. That said, in practice, only the first match would be effective, so this might not be a real issue.  
- `refs/tags/@liam-hq/cli@0.4.1` could also match something like `refs/tags/@liam-hq/cli@0.4.10`. However, considering that version numbers are properly incremented and only the highest value is being searched, this might not be a significant issue either.

Reply:
Thanks for the explanation. I did not handle the tagCommit properly.🙏
 I removed it in the current code!